crates/net/eth-wire-types/src/version.rs

```diff
@@ -1,8 +1,11 @@
 //! Support for representing the version of the `eth`
 
-use std::str::FromStr;
+use bytes::BufMut;
+use std::{fmt, str::FromStr};
 
+use alloy_rlp::{Decodable, Encodable, Error as RlpError};
```

Comment:
```suggestion
use std::{fmt, str::FromStr};

use bytes::BufMut;
use alloy_rlp::{Decodable, Encodable, Error as RlpError};
```

prelude should at best follow pattern:

std or core libraries

other libraries

local libraries (crate, super or a module)

no need to change everywhere though in this pr